tests/migrations/test_commands.py

```diff
@@ -1098,6 +1105,15 @@ def test_sqlmigrate_unrepresentable(self):
             ],
         )
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations"})
+    def test_sqlmigrate_transaction_keywords_not_colorized(self):
+        out = io.StringIO()
+        with mock.patch(
+            "django.core.management.color.supports_color", lambda *args: True
+        ):
+            call_command("sqlmigrate", "migrations", "0001", stdout=out, no_color=False)
+        self.assertNotIn("\x1b", out.getvalue())
```

Comment:
This behavior was tested incidentally as part of the two other tests I touched in this diff, but only on colorized terminals, not on CI. So I factored out this test case, ensured it [fails](https://github.com/jacobtylerwalls/django/actions/runs/11999629448/job/33447768162) on CI without my change, and set the existing tests to `no_color=True` as most other tests do in similar situations.